### PR TITLE
feature-001

### PR DESCRIPTION
**Pull request by nguyenhoangnamkg**
Feature 001 is done and ready for merging

### CODE REVIEW

File: customer/Dockerfile
Comment (hoangnam1992kiengiang) [inline customer/Dockerfile:1]: Are you sure?

**Review by hoangnam1992kiengiang — COMMENTED** [review]
Please resolve this first

File: customer/.env.dev
Comment (hoangnam1992kiengiang) [inline customer/.env.dev:1]: is the connection details correct?

Comment (nguyenhoangnamkg) [reply customer/.env.dev:1]: Yes, correct

File: customer/src/config/index.js
Comment (hoangnam1992kiengiang) [inline customer/src/config/index.js:1]: How about this, can you add more comments?

**Review by hoangnam1992kiengiang — APPROVED** [review]

**Review by hoangnam1992kiengiang — APPROVED** [review]

**Review by hoangnam1992kiengiang — APPROVED** [review]

**Pull request merged**